project/src/test/java/com/google/sps/servlets/QueryServletTest.java

```diff
@@ -134,21 +134,18 @@ public void getRequest_userPreferencesForwardedToFetcher() throws Exception {
     when(REQUEST.getParameter("rating")).thenReturn("4");
     when(REQUEST.getParameter("price")).thenReturn("3");
     when(REQUEST.getParameter("open")).thenReturn("1");
-    when(REQUEST.getParameter("location")).thenReturn("00.00000000,00.00000000");
+    when(REQUEST.getParameter("location")).thenReturn("35.35000000,30.00000000");
```

Comment:
Are all those zeros necessary? What happens if you just write .thenReturn("35.35,30")?